frontend/src/views/Org/MembersPage/components/OrgMembersTab/components/OrgMembersSection/OrgMembersTable.tsx

```diff
@@ -141,10 +141,10 @@ export const OrgMembersTable = ({ handlePopUpOpen, setCompleteInviteLink }: Prop
     () =>
       members?.filter(
         ({ user: u, inviteEmail }) =>
-          u?.firstName?.toLowerCase().includes(searchMemberFilter) ||
-          u?.lastName?.toLowerCase().includes(searchMemberFilter) ||
-          u?.email?.toLowerCase().includes(searchMemberFilter) ||
-          inviteEmail?.includes(searchMemberFilter)
+          u?.firstName?.toLowerCase().includes(searchMemberFilter.toLowerCase()) ||
```

Comment:
Instead of changing each searchMemberFilter, you can just change all to be lower case via `setSearchMemberFilter`

Reply:
Hi @maidul98  , thank you for your review, When we use `setSearchMemberFilter` to update the state, such as `setSearchMemberFilter(e.target.value.toLowerCase())`, the user is unable to search in uppercase, even when they attempt to do so; the uppercase word is converted to smaller case in the search field itself.

Reply:
Correct. Agreed. The other one user will be confused because when they type the casing gets converted